lib/config-helper.js

```diff
@@ -0,0 +1,360 @@
+
+var fs = require('fs-extra');
+var pathExtra = require('path-extra');
+var configWatcher = require('./config-watcher');
+
+module.exports = function(){
+
+  var config;
+  var home = '';
+  var mainAppHost = 'localhost';
+  var mainAppPort = 19104;
+  var appsPort = 18001;
+  //var loadedPlugins = {};
+
+  var configHelpers = {
+
+    watcher: null,
+
+    /**
+     * Initialize config instance
+     */
+    init: function (cozyHome) {
+      fs.mkdirsSync(cozyHome);
+      configHelpers.setHomePath(cozyHome);
+      configHelpers.copyDependency('pouchdb');
+      configHelpers.createConfigFile();
+      configHelpers.watcher = configWatcher(configHelpers.getConfigPath());
+      return configHelpers.loadConfigFile();
+    },
+
+    /**
+     * Sets Home directory path
+     */
+    setHomePath: function (h) {
```

Comment:
Could you avoid single letter variables and only use full variable name instead?